### PR TITLE
General enhancements

### PR DESCRIPTION
**Pull request by ljmotta**
Hi, I've made some general enchancements.

On this PR:
 - Use the wrapField on the `BoolField` and `RadioField` components;
 - Add the `isRequired` props on the `FormGroup`, and add tests;
 - Remove the placeholder `isDisabled` on the `SelectField` component, and use React hooks on the callbacks;
 - Remove margin on the `Card` component;
 - Add `sourceMap` option to improve debug;
 - Add validation on the `NumField` component;

### CODE REVIEW

**Review by wtrocki — APPROVED** [review]
Amazing contribution

**Comment by wtrocki** [comment]
@kingsleyzissou do you mind taking look and releasing?

File: src/SelectField.tsx [hunk 1]
```diff
@@ -91,57 +93,70 @@ function isSelectOptionObject(
...
-    if (fieldType !== Array) {
-      return parsedSelection !== '' ? parsedSelection : '';
-    }
-
-    if (Array.isArray(selected)) {
-      if (selected.includes(parsedSelection)) {
-        return selected.filter((s) => s !== parsedSelection);
+  const [selected, setSelected] = useState<string | string[]>([]);
+
+  const parseInput = useCallback(
```
Comment (kingsleyzissou) [inline src/SelectField.tsx:99]: So happy with this, this is something that I was meaning to come back to, but haven't had the chance to do.

**Review by kingsleyzissou — APPROVED** [review]
Brilliant stuff! I'll sort out a release for this.

**Pull request merged**